packages/apollo-codegen-swift/src/language.ts

```diff
@@ -62,7 +62,15 @@ export class SwiftGenerator<Context> extends CodeGenerator<
   }
 
   multilineString(string: string) {
-    this.printOnNewline(`"${escapedString(string)}"`);
+    this.printNewline();
+    this.print(`"""`);
+
+    string.split("\n").forEach(line => {
+      this.printNewline();
+      this.print(line);
```

Comment:
As mentioned at the top-level discussion, this isn't handling any GraphQL multiline strings (which use the same `"""` token). Also looking at this right now, it's not going to handle any backslash-escapes in the GraphQL source either.

Reply:
Yeah...blargh. I might take another pass at this as a comment for readability, but leave the actual `operationDefinition` as a one-liner for the reasons you outlined.